src/main/java/com/minecolonies/coremod/placementhandlers/PlacementHandlers.java

```diff
@@ -6,6 +6,8 @@
 import com.minecolonies.api.util.ItemStackUtils;
 import com.minecolonies.coremod.blocks.BlockMinecoloniesRack;
 import com.minecolonies.coremod.blocks.BlockSolidSubstitution;
+import com.minecolonies.coremod.blocks.BlockWaypoint;
+import com.minecolonies.coremod.blocks.ModBlocks;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.blocks.ModBlocks'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)